google/cloud/bigtable/rpc_retry_policy.h

```diff
@@ -27,34 +28,31 @@ namespace cloud {
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
 namespace internal {
+/**
+ * Returns `true` for gRPC error messages, with status code: `kInternal`, that
+ * are known to be retryable across all services.
+ */
+bool IsTransientInternalError(Status const& status);
+
 /// An adapter to use `grpc::Status` with the `google::cloud::*Policies`.
 struct SafeGrpcRetry {
-  static inline bool IsTransientFailure(google::cloud::StatusCode code) {
-    return code == StatusCode::kAborted || code == StatusCode::kUnavailable ||
-           code == StatusCode::kDeadlineExceeded;
-  }
-
   static inline bool IsOk(google::cloud::Status const& status) {
     return status.ok();
   }
   static inline bool IsTransientFailure(google::cloud::Status const& status) {
-    return IsTransientFailure(status.code());
+    auto const code = status.code();
+    return code == StatusCode::kAborted || code == StatusCode::kUnavailable ||
+           code == StatusCode::kDeadlineExceeded ||
+           IsTransientInternalError(status);
```

Comment:
Can't you just call `google::cloud::internal::IsTransientInternalError()` rather than having a bigtable copy of it?

Reply:
Well I was thinking 2 things.
1. I wanted to avoid discussion about whether the function should be defined in `g/c/i/retry_policy.h`. Maybe it belongs in `g/c/grpc_error_delegate.h`. (This has backfired, as we are now discussing whether the function belongs in `g/c/bt/rpc_retry_policy.h` 😆)
2. Maybe there will be a transient internal error for a service that should not be retried by the others. Then refactoring into common might not be ideal. (Currently, this is not the case, which is why I think the function does belong in common)

Anyway, the function has been moved. `spanner/` is untouched. PTAL.